src/main/java/org/andersen/homework/controller/BusTicketController.java

```diff
@@ -0,0 +1,60 @@
+package org.andersen.homework.controller;
+
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.andersen.homework.model.dto.ticket.bus.BusTicketDto;
+import org.andersen.homework.model.dto.ticket.bus.BusTicketSaveDto;
+import org.andersen.homework.model.dto.ticket.bus.BusTicketUpdateDto;
+import org.andersen.homework.service.ticket.BusTicketService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/bus-ticket")
+public class BusTicketController {
+
+  private final BusTicketService busTicketService;
+
+  @GetMapping("/by-id")
+  public BusTicketDto getTicketById(@RequestParam UUID id) {
+    return busTicketService.getById(id);
+  }
+
+  @GetMapping("/by-client-id")
+  public List<BusTicketDto> getTicketByClientId(@RequestParam UUID clientId) {
```

Comment:
Looks like we don't need 2 separate endpoint for getTicketById and getTicketByClientId
Let's separate it on service level